### DISCUSSION TITLE
SetSourceIgnore与IUpdate.IgnoreColumns怎么同时使用？

### DISCUSSION THREAD

**Discussion by TinyMaD** [dotnetcore/FreeSql#921] [Q&A]
我有一个需求，忽略为null的值，以及几个特定的字段，怎么办？

目前只想到一个方法：在更新前，把特定的字段的值全部赋值为null

有没有其他更优雅一点的方法

**Comment by 2881099** [comment]
只能二选一哦

**Reply by TinyMaD** [reply]
谢谢回复，已经解决，在特定忽略更新的字段上加的[Column(CanUpdate = false)]
